src/ErrorHandler/PsrErrorHandler.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types = 1);
+
+namespace Apitte\Core\ErrorHandler;
+
+use Apitte\Core\Exception\Runtime\SnapshotException;
+use Apitte\Core\Utils\Helpers;
+use Psr\Log\LoggerInterface;
+use Throwable;
+
+class PsrErrorHandler implements IErrorHandler
+{
+
+	/** @var LoggerInterface */
+	private $logger;
+
+	/** @var bool */
+	private $catchException = false;
+
+	public function __construct(LoggerInterface $logger)
+	{
+		$this->logger = $logger;
+	}
+
+	public function setCatchException(bool $catchException): void
+	{
+		$this->catchException = $catchException;
+	}
+
+	public function handle(Throwable $throwable): void
+	{
+		// Log exception
+		$this->logger->error($throwable->getMessage(), ['exception' => Helpers::throwableToArray($throwable)]);
```

Comment:
Proc se nepreda cela exception?

https://github.com/Seldaek/monolog/issues/528#issuecomment-156059577

Reply:
Myslel jsem, že je to monolog-specific, upraveno.